web/pages/requests.tsx

```diff
@@ -13,10 +15,11 @@ interface RequestsProps {
   page: number;
   from: number;
   to: number;
+  properties: string[];
```

Comment:
if there are no custom properties this will default to an empty array right?

Reply:
That's correct, we query the DB for all the properties, each row in the response is a property, so if no custom properties the response has no rows and this is empty

Reply:
this comment actually helped me uncover a bug when there are no properties which I now fixed, so thanks!